### ISSUE TITLE
Create error handling for invalid parameters.json

### ISSUE THREAD

**Issue by jongogogo** [awslabs/aws-config-rdk#100]
My `rdk deploy -f --all` cmd failed, with the entire pipeline due to one `parameters.json`.

The error is cryptic. Ideally I would add a explicit warning about the rule not being deploy (rule name, file, line/collum, error message).

Current error FYI
```
[Container] 2018/08/28 00:03:47 Running command rdk deploy -f --all > ../result.txt
Traceback (most recent call last):
  File "/usr/local/bin/rdk", line 33, in <module>
    return_val = my_rdk.process_command()
  File "/usr/local/lib/python3.6/site-packages/rdk/rdk.py", line 68, in process_command
    exit_code = method_to_call()
  File "/usr/local/lib/python3.6/site-packages/rdk/rdk.py", line 538, in deploy
    function_template = self.__create_function_cloudformation_template()
  File "/usr/local/lib/python3.6/site-packages/rdk/rdk.py", line 1825, in __create_function_cloudformation_template
    params = self.__get_rule_parameters(rule_name)
  File "/usr/local/lib/python3.6/site-packages/rdk/rdk.py", line 1414, in __get_rule_parameters
    my_json = json.load(parameters_file)
  File "/usr/local/lib/python3.6/json/__init__.py", line 299, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/local/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/lib/python3.6/json/decoder.py", line 355, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 10 column 5 (char 326)
```

**Comment by jongogogo** [comment]
Issue #100 Boom!

**Comment by michaelborchert** [comment]
Addressed in PR #144.
